site2/docs/functions-quickstart.md

```diff
@@ -1,20 +1,20 @@
 ---
-id: functions-quickstart
-title: Get started with Pulsar Functions
-sidebar_label: Get started
+id: functions-quickstart  
+title: Get started with Pulsar Functions  
+sidebar_label: Get started  
 ---
 
-This tutorial will walk you through running a [standalone](reference-terminology.md#standalone) Pulsar [cluster](reference-terminology.md#cluster) on your machine and then running your first Pulsar Functions using that cluster. The first function will run in local run mode (outside your Pulsar [cluster](reference-terminology.md#cluster)), while the second will run in cluster mode (inside your cluster).
+This tutorial walks you through running a [standalone](reference-terminology.md#standalone) Pulsar [cluster](reference-terminology.md#cluster) on your machine, and then running your first function using that cluster. The first function runs in local run mode (outside your Pulsar [cluster](reference-terminology.md#cluster)), while the second runs in cluster mode (inside your cluster).
```

Comment:
> ...and then running your first function using that cluster....

1. Should we explicitly use "Pulsar Function" the first time we refer to it.

2. Should we capitalize "f" for "function" here because we are referring to a "Pulsar Function" and not a generic function.  The Azure Functions documentation does this.

Reply:
Sure, if it refers to "Pulsar Functions", I'll update it. Thank @jerrypeng for your advice.